### DISCUSSION TITLE
Reverse backup for pb_data

### DISCUSSION THREAD

**Discussion by tigawanna** [pocketbase/pocketbase#2547] [Q&A] [answered]
With the backup option will there be support for an API to push a pb_data directory to pocketbase programmatically , would be useful for seeding a prod instance with the dev pb_data

**Comment by ganigeorgiev** [comment, marked as answer]
You mean uploading manually a zip backup?

Technically we can allow it but I'm afraid it may get misused by uploading some random files or `pb_data` with invalid structure. It'll also require more comprehensive checks on restore when extracting the archive (_we currently check only the minimum, aka. whether there is a data.db file_).

I've added it my todo to consider it in the future, but for now I'm not planning working on it.

**Reply by tigawanna** [reply]
Yup uploading the zipped directory which would be an easier way of file migration I'd naively guess that a backup could be done before attempting to load in the uploaded files and it would auto roll back if turned out corrupted since the backup API already exists. I would understand if this stayed low priority.
Thanks for the great work
